### ISSUE TITLE
New release does not work on PHP 7.4

### ISSUE THREAD

**Issue by flack** [cocur/slugify#328]
Running the latest release on PHP 7.4 results in the following error:

```
ParseError: syntax error, unexpected '|', expecting variable (T_VARIABLE)

cocur/slugify/src/Slugify.php:100
```

If you decided not to support PHP 7.4 anymore, that is fine by me, but please adjust `composer.json` accordingly, otherwise existing installations break for no reason:

https://github.com/cocur/slugify/blob/af8e6ee771458bf885f7457807b5ff9bad8743cb/composer.json#L22

**Comment by florianeckerstorfer** [comment]
@flack I'm very sorry, I have released v4.4.1 and v4.5.1 to replace the two releases with the broken version constraint.

**Comment by flack** [comment]
@florianeckerstorfer awesome, thanks!
